tests/container_disk_test.go

```diff
@@ -100,19 +97,21 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting and stopping the same VirtualMachineInstance", func() {
 		Context("with ephemeral registry disk", func() {
 			It("[test_id:1463][Conformance] should success multiple times", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				vmi := libvmi.NewCirros()
 				num := 2
 				for i := 0; i < num; i++ {
 					By("Starting the VirtualMachineInstance")
 					obj, err := virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(util.NamespaceTestDefault).Body(vmi).Do(context.Background()).Get()
 					Expect(err).To(BeNil())
 					tests.WaitForSuccessfulVMIStart(obj)
 
+					createdVMI := obj.(*v1.VirtualMachineInstance)
 					By("Stopping the VirtualMachineInstance")
-					_, err = virtClient.RestClient().Delete().Resource("virtualmachineinstances").Namespace(vmi.GetObjectMeta().GetNamespace()).Name(vmi.GetObjectMeta().GetName()).Do(context.Background()).Get()
+					_, err = virtClient.RestClient().Delete().Resource("virtualmachineinstances").
+						Namespace(createdVMI.Namespace).Name(createdVMI.Name).Do(context.Background()).Get()
 					Expect(err).To(BeNil())
 					By("Waiting until the VirtualMachineInstance is gone")
-					tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
+					tests.WaitForVirtualMachineToDisappearWithTimeout(createdVMI, 120)
```

Comment:
These changes do not seem to be related to the commit.

Reply:
They are related, I'm not sure why you're saying they're not, can you elaborate?